api/src/logger.ts

```diff
@@ -2,10 +2,14 @@
 const logLevels = ["debug", "info", "warn", "error"] as const;
 type LogLevel = (typeof logLevels)[number];
 
+const isDev = process.env.npm_lifecycle_event === "dev";
+
 // Determine the current log level from the environment variable or default to "info"
-const currentLogLevel: LogLevel = isLogLevel(process.env.FPX_LOG_LEVEL)
-  ? process.env.FPX_LOG_LEVEL
-  : "info";
+const currentLogLevel: LogLevel = isDev
+  ? "debug"
+  : isLogLevel(process.env.FPX_LOG_LEVEL)
+    ? process.env.FPX_LOG_LEVEL
+    : "info";
```

Comment:
Could still be nice to turn off debug logging in dev, so we can simulate the end user terminal output

```suggestion
const defaultLogLevel: LogLevel = isDev ? "debug" : "info"
const currentLogLevel: LogLevel = isLogLevel(process.env.FPX_LOG_LEVEL)
    ? process.env.FPX_LOG_LEVEL
    : defaultLogLevel;
```